### DISCUSSION TITLE
如何部署在github pages上面

### DISCUSSION THREAD

**Discussion by lilimu996** [hugo-fixit/FixIt#499] [Q&A] [answered]
在本地可以启动成功，然后我使用github actions把它部署到github pages上面后没有样式，是还需要修改一些配置吗
本地启动是这个样子
![image](https://github.com/user-attachments/assets/1ec4e3eb-553d-4ec0-987b-8b705ffd46f3)
我按照教程把整个public文件夹上传的github后，进去是这个样子
![image](https://github.com/user-attachments/assets/3078c8e1-d187-42b5-8997-00e12fd244d8)

**Comment by Lruihao** [comment, marked as answer]
本地和部署后的路径不对，部署的仓库名应该是 `你的用户名.github.io`

**Reply by lilimu996** [reply]
太感谢了
